packages/indexer/src/indexers/autocrat/autocrat-proposal-indexer.ts

```diff
@@ -615,3 +619,82 @@ async function insertAssociatedAccountsDataForProposal(
       .execute()
   );
 }
+
+async function calculateUserPerformance(onChainProposal: ProposalAccountWithKey) {
+    // calculate performance
+    const [ proposal ] = await usingDb(db => {
+      return db
+        .select()
+        .from(schema.proposals)
+        .where(eq(schema.proposals.proposalAcct, onChainProposal.publicKey.toString()))
+        .leftJoin(schema.daos, eq(schema.proposals.daoAcct, schema.daos.daoAcct))
+        .leftJoin(schema.tokens, eq(schema.daos.baseAcct, schema.tokens.mintAcct))
+        .limit(1)
+        .execute()
+    })
+
+    const { proposals, tokens } = proposal
+
+    const orders = await usingDb(db => {
+      return db
+        .select()
+        .from(schema.orders)
+        .where(inArray(schema.orders.marketAcct, [proposals.passMarketAcct, proposals.failMarketAcct]))
+        .execute()
+    })
+
+    let actors = <any>{}
+    
+    orders.forEach(o => {
+      const actor = o.actorAcct
+      if (!actors[actor]) {
+        actors[actor] = {
+          tokensBought: 0,
+          tokensSold: 0,
+          volumeBought: 0,
+          volumeSold: 0
+        }
+      }
+
+      const totals = actors[actor]
+      const orderAmount = PriceMath.getHumanAmount(new BN(o.filledBaseAmount), tokens?.decimals)
+      const price = o.quotePrice * orderAmount
```

Comment:
Just a nit, this is volume or amount where as size is filledBaseAmount. Size * price = amount.

Reply:
`orderAmount` is size and `price` is order value. Maybe I should change variable names.